avm/res/network/virtual-network-gateway/main.bicep

```diff
@@ -55,7 +59,7 @@ param vpnGatewayGeneration string = 'None'
   'ErGw2AZ'
   'ErGw3AZ'
 ])
-param skuName string
+param skuName string = (gatewayType == 'VPN') ? 'VpnGw1AZ' : 'ErGw1AZ'
```

Comment:
Hey @sebassem thanks for this update. Since a default value is provided for this parameter, we should also change the parameter type to Optional in the description metadata and regenerate the readme.

@fabmas could you please follow this review next?

Thanks!

Reply:
Resolved